Common/include/config_structure.hpp

```diff
@@ -828,9 +847,26 @@ class CConfig {
   *default_htp_axis,          /*!< \brief Default HTP axis for the COption class. */
   *default_ffd_axis,          /*!< \brief Default FFD axis for the COption class. */
   *default_inc_crit,          /*!< \brief Default incremental criteria array for the COption class. */
-  *default_body_force;        /*!< \brief Default body force vector for the COption class. */
+  *default_extrarelfac;       /*!< \brief Default extra relaxation factor for NRBC in the COption class. */
+  unsigned short nSpanWiseSections; /*!< \brief number of span-wise sections */
+  unsigned short nSpanMaxAllZones; /*!< \brief number of maximum span-wise sections for all zones */
```

Comment:
Once we accept this pull request we need to start thinking about how to unify wing geometrical analysis with blade geometrical analysis.